doc/advanced_setup.rst

```diff
@@ -103,6 +103,29 @@ runtime accordingly.
 If you use another Python setup and you encounter some difficulties please
 report them on the MNE mailing list or on github to get assistance.
 
+Installing Mayavi
+^^^^^^^^^^^^^^^^^
+
+If you have Anaconda installed (recommended), the easiest way to install `mayavi` is to do::
+
+    conda install mayavi
```

Comment:
one reason to add `>>>` to the Python code snippets is the same reason you probably should add a `$` prefix to these lines -- to make it clearer which language is being used.